wgpu-core/src/device/queue.rs

```diff
@@ -1468,8 +1448,18 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                 }
             }
 
```

Comment:
Note that we essentially have to inline `pre_submit` here, since we're using a reference into dst_textures and pre_submit would need `&mut self` over the whole structure. We clear dst_textures after submit instead.